project/Build.scala

```diff
@@ -249,6 +252,10 @@ object Build {
           "-encoding", "utf8"
       ),
 
+      scalastyleCheck := Def.taskDyn {
+        ScalastylePlugin.autoImport.scalastyle.in(Test).toTask("")
```

Comment:
Does running this in the `Test` configuration also check the normal code? (Our `Jenkinsfile` suggests otherwise, but it might be over-checking).

Reply:
Also, write this:

```scala
(scalastyle in Test).toTask("")
```

And import `scalastyle` for the entire file?

Reply:
ahh yeah, I was originally trying to keep the name of the task `scalastyle` so was trying to avoid the double naming, but that didn't work out, I'll update the import appropriately.  I'll double check the `taskDyn` that was cargo cult on my part from examples I could find.

Reply:
`taskDyn` is required here, nothing runs if `Def.task`

Reply:
It looks like both in Test and in Compile are required :/ trying to figure out if I need to have two tasks or if I can put them both in the same one

Reply:
> nothing runs if Def.task

Nothing runs or it fails to compile? If it doesn't run, you might be creating a dangling value. Try:

```scala
scalaStyleCheck := (scalastyle in Test).toTask("")

// or

scalaStyleCheck := Def.task {
  (scalastyle in Test).toTask("").value
}.value
```

Reply:
Adding the extra `.value` seemed to do the trick.

Full code got updated to:

```
      scalastyleCheck := Def.task {
        val _ = (scalastyle in Compile).toTask("").value
        (scalastyle in Test).toTask("").value
      }.value,
```